src/api/item.ts

```diff
@@ -74,14 +75,16 @@ export const getAccessibleItems = async (
       .then(({ data }) => data),
   );
 
+type AllOrNothing<T> = T | Partial<Record<keyof T, undefined>>;
 export type PostItemPayloadType = Partial<DiscriminatedItem> &
   Pick<DiscriminatedItem, 'type' | 'name'> & {
     parentId?: UUID;
-  };
-// payload = {name, type, description, extra}
+  } & AllOrNothing<Pick<ItemGeolocation, 'lat' | 'lng'>>;
```

Comment:
Following the suggestion above this would be:
```suggestion
  } & { geolocation?: Pick<ItemGeolocation, 'lat' | 'lng'>};
```